src/libs/kata-sys-util/src/mount.rs

```diff
@@ -828,7 +834,7 @@ mod tests {
             Err(Error::InvalidPath(_))
         ));
 
-        bind_mount_unchecked(tmpdir2.path(), tmpdir.path(), true).unwrap();
+        bind_mount_unchecked(tmpdir2.path(), tmpdir.path(), true, "slave").unwrap();
```

Comment:
I think you could just pass `MsFlags::MS_SLAVE ` as parameter, so you don't need to match the flag type in `bind_mount_unchecked`

Reply:
I agree, since it is not parsed from a configuration.

Reply:
sure, fixed!